### ISSUE TITLE
Favicons in Safari 12.0 tabs not correct

### ISSUE THREAD

**Issue by Zillion01** [RealFaviconGenerator/realfavicongenerator#400]
In Safari 12.0 (MacOS Mojave) favicons can be activated in preferences for the tabs. They get displayed in a mono kind color, or colors. See image in attachment.
![favicons](https://user-images.githubusercontent.com/1055822/47388070-87596a00-d711-11e8-82e9-7a791b2190d1.png)

**Comment by phbernard** [comment]
Got it. The classic favicon is also shown as the pinned tabs icon, which is probably what 99% people expect instead of the monochromatic mask icon.

Guess it's time to remove macOS Safari mask icon.

Thank you for reporting!

**Comment by ikesau** [comment]
Has there been any update on this? I'm getting just the mask on Version 12.1.1

![image](https://user-images.githubusercontent.com/11844404/60732760-e2866200-9f18-11e9-9fbf-eecc4d530ece.png)

**Comment by alTach** [comment]
plugin cannot generate this icon for safari tab (safari-pinned-tab.svg). Generated svg is black;
![image](https://user-images.githubusercontent.com/15833998/65154644-8cfc2480-da34-11e9-85b5-55fb2c630a67.png)

**Comment by henryiii** [comment]
The full color icon looks much better in the current safari than the mask icon. It would be nice to make it optional in the UI, at least.

**Comment by schmod** [comment]
Yeah, it's probably worth at least mentioning that the mask icon appears to be unnecessary in Safari 12 and over.

**Comment by lukepighetti** [comment]
Problem is, if I delete the mask icon, there is no favicon for safari to use 🤔
